pkg/kn/commands/flags/traffic.go

```diff
@@ -0,0 +1,56 @@
+// Copyright © 2019 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flags
+
+import (
+	"github.com/spf13/cobra"
+)
+
+type Traffic struct {
+	RevisionsPercentages []string
+	RevisionsTags        []string
+	UntagRevisions       []string
+}
+
+func (t *Traffic) Add(cmd *cobra.Command) {
+	cmd.Flags().StringSliceVar(&t.RevisionsPercentages,
+		"traffic",
+		nil,
+		"Set traffic distribution (format: --traffic revisionRef=percent) where revisionRef can be a revision or a tag or '@latest' string "+
+			"representing latest ready revision. This flag can be given multiple times with percent summing up to 100%.")
+
+	cmd.Flags().StringSliceVar(&t.RevisionsTags,
+		"tag",
+		nil,
+		"Set tag (format: --tag revisionRef=tag) where revisionRef can be a revision or '@latest' string representing latest ready revision. "+
+			"This flag can be specified multiple times.")
+
+	cmd.Flags().StringSliceVar(&t.UntagRevisions,
+		"untag",
+		nil,
+		"Untag revision (format: --untag tag). This flag can be spcified multiple times.")
```

Comment:
Don't use `tag` as the tag for the `--untag` example. That confuses things IMO since tag is a flag. Would make more sense to use a more concrete example, e.g., `v0.1` or similar?

Reply:
how about changing the all `tagName`? 
(the `format` in flag description is field reference in traffic block)